solution/deps.yaml

```diff
@@ -20,7 +20,7 @@ cloudserver:
   sourceRegistry: registry.scality.com/cloudserver
   dashboard: cloudserver-dashboards
   image: cloudserver
-  tag: 8.4.3
+  tag: 8.4.4
```

Comment:
if you bump cloudserver to 8.4.4 (and get slow listing), you must bump backbeat as well so it supports the format (v1) of newly created buckets .

backbeat should be at least 8.3.9.